source/_integrations/roborock.markdown

```diff
@@ -18,7 +18,7 @@ ha_integration_type: integration
 
 The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while continuing to use the Roborock app.
 
-In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access to set up and device configuration using the Roborock app.
+In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support using the Roborock app.
```

Comment:
```suggestion
In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides a more complete support but requires that the device is configured using the Roborock app.
```
Maybe something like this? I'm not sure about the proper wording, see my other comment. If the device can still be controlled using the mi app even after it has been configured using the roborock one, this suggestion is irrelevant.

Reply:
But this Integration does not feature a more complete support. The contrary is the case: This integration is lacking some key features.

The point of the original text is, that using this integration allows to use the **Roborock app** and it is what allows more features than using the Xiaomi Home app.

Reply:
Let's remove the whole line.

```suggestion
```